### ISSUE TITLE
Environment radio buttons are re-enabled when you start sim and don't carry over when loaded

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/natural-selection#296]
**Device**
Mac and Dell
**OS**
MacOS 11 and Win 10
**Browser**
Firefox and Chrome
**Problem Description**
For https://github.com/phetsims/QA/issues/662. Seen with this element, but may happen with other elements as well. Seems similar to https://github.com/phetsims/gravity-and-orbits/issues/394. `naturalSelection.introScreen.view.environmentPanel.environmentRadioButtonGroup.enabledProperty`
When you disable the above element, pressing "Add a Mate" reenables the buttons, both in studio or a launched sim. In addition, if you save a sim with that element disabled, reenable it in studio, then "Load" that saved file, the property will not be disabled.
**Visuals**
![reenable2](https://user-images.githubusercontent.com/41024075/124198759-95e5ab00-da8e-11eb-8c58-fb2a9256ff78.gif)
![reenable](https://user-images.githubusercontent.com/41024075/124198766-98480500-da8e-11eb-91bf-420df63fbffe.gif)

**Comment by pixelzoom** [comment]
The relevant code is in EnvironmentPanel.js:

```js
    // Simulation mode determines which UI components are enabled. unlink is not necessary.
    model.simulationModeProperty.link( simulationMode => {
      environmentRadioButtonGroup.enabledProperty.value = ( simulationMode !== SimulationMode.COMPLETED );
    } );
```

The problem is that the sim controls `environmentRadioButtonGroup.enabledProperty`.  When a simulation has "completed" (bunnies have taken over the world, or all bunnies have died), the `environmentRadioButtonGroup.enabledProperty` is set to false, so that the user can review results, but not change the environment. 

There is no way to know whether `environmentRadioButtonGroup.enabledProperty` was set via Studio/PhET-iO.  Ether the sim can control a Property, or Studio/PhET-iO can control it, not both.  So there options are:

(1) Set `environmentRadioButtonGroup.enabledProperty` to read-only, so that it can't be changed by Studio/PhET-iO.  Instructional designers will not be able to disabled the buttons, but they can hide them.

(2) Remove the above code, so that the sim never changes `environmentRadioButtonGroup.enabledProperty`.   The student will be able to change the environment when reviewing results, which is not ideal, but not tragic.

This is a PhET-iO design issue, and the associated QA issue is top priority. So assigning to @amanda-phet with top priority.

**Comment by amanda-phet** [comment]
I agree with @pixelzoom . Option (1) is my preference.

**Comment by pixelzoom** [comment]
Option (1) has been implemented, patched in master and 1.4 branches. 

Ready for testing in the next 1.4 RC.

**Comment by KatieWoe** [comment]
Looks ok in rc.2
